### ISSUE TITLE
Issue with Ratchet Deadlocked (PS2)

### ISSUE THREAD

**Issue by Peapodman** [isJuhn/KAMI#22]
Hi I'm running Ratchet Deadlocked on PCSX2 v1.7.0-dev-1177-gedeb0d7bd and everything was working really well until I reached the second planet in the game (Catacrom Four) and it suddenly stopped working. So I went back to the first planet (DreadZone Station) and it worked again. Then when I went back to Catacrom Four it still wouldn't work. So I guess the issue really is with Catacrom Four? I can't progress without using the mouse so I can't test this much further. 

I've tried restarting PCSX2 and KAMI and the problem persisted - I'm not much of a coding guy so I'm not sure what other tests to run. 

Otherwise it was working really well and you've done a cracking job, thanks! I'm really excited to get into this in the future!

Cheers,
Ben

**Comment by isJuhn** [comment]
yes the ps2 single player is problematic, probably won't look into it again for a while.
